health-services/project-factory/src/server/utils/excelUtils.ts

```
@@ -165,8 +235,121 @@ function addDataToSheet(
+      formula: rowFormula
+    };
+  }
 }
 
+// Utility function to get column letter from index
+function getColumnLetter(index: number): string {
+  let letter = '';
+  while (index > 0) {
+    const remainder = (index - 1) % 26;
+    letter = String.fromCharCode(65 + remainder) + letter;
+    index = Math.floor((index - 1) / 26);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)